src/test/main_tests.cpp

```diff
@@ -101,13 +101,11 @@ BOOST_AUTO_TEST_CASE(subsidy_limit_test)
     // 11th), decreasing the total monetary supply by 0.0693 ZEC. If the
     // transaction output field is widened, this discrepancy will become smaller
     // or disappear entirely.
```

Comment:
I don't know what is referred to by "If the transaction output field is widened". This may be a design choice that was never taken (and is now unlikely to be taken); if so then this sentence should probably be removed.